string game_type
StepDef[] steps
